src/docregistry/registry.ts

```diff
@@ -400,7 +400,7 @@ class DocumentRegistry {
   widgetExtensions(widgetName: string): IIterator<DocumentRegistry.IWidgetExtension<Widget, DocumentRegistry.IModel>> {
     widgetName = widgetName.toLowerCase();
     if (!(widgetName in this._extenders)) {
-      return EmptyIterator.instance;
+      return empty() as IIterator<DocumentRegistry.IWidgetExtension<Widget, DocumentRegistry.IModel>>;
```

Comment:
`return empty<IIterator<DocumentRegistry.IWidgetExtension<Widget, DocumentRegistry.IModel>>>()`

and `DocumentRegistry.IWidgetExtension<Widget, DocumentRegistry.IModel>` should really be type aliased to something shorter :)